brats/src/amazing_logic.py

```diff
@@ -0,0 +1,38 @@
+"""Logic file"""
+import os
+from shutil import copyfile
+from dotenv import dotenv_values
+from src.utils.utilities import helper
+
+config = dotenv_values(".env")
+APPLICATION_NAME = config["APPLICATION_NAME"]
+APPLICATION_VERSION = config["APPLICATION_VERSION"]
+INPUT_FOLDER = config["INPUT_FOLDER"]
```

Comment:
Read INPUT_FOLDER and OUTPUT_FOLDER from --data_dir and --results_dir only; and taken them out of the .env file